src/vcpkg/versions.cpp

```diff
@@ -409,4 +409,98 @@ namespace vcpkg
 
         return static_cast<VerComp>(Util::range_lexcomp(a.identifiers, b.identifiers, uint64_comp));
     }
+
+    StringView normalize_external_version_zeros(StringView sv)
```

Comment:
Should try_parse_blah just do this always?

Reply:
No, see tools.cpp:47